tests/unit_tests/dynamics/collisions/test_kernels.py

```diff
@@ -69,6 +71,9 @@ def test_simple_geometric_same_size(volume):
         _Indicator = builder.particulator.PairIndicator
         output = _PairwiseStorage.from_ndarray(np.zeros_like(volume))
         is_first_in_pair = _Indicator(length=volume.size)
+        is_first_in_pair.indicator = builder.particulator.Storage.from_ndarray(
+            np.asarray([True, False])
+        )
 
         # act
         sut(output, is_first_in_pair=is_first_in_pair)
```

Comment:
I appreciate all the cleaning up of these test files as well!

Reply:
No problem! :)